prototyping/Search.py

```diff
@@ -0,0 +1,50 @@
+from MockArticle import Article
```

Comment:
[nit] Consider importing `MockArticle` as a whole and then use `MockArticle.Article`

Reply:
I'm sorry but why? I myself think it looks better this way. As far as the style guide goes it says either way is okay https://google.github.io/styleguide/pyguide.html#224-decision